app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,47 @@
+class OrdersController < ApplicationController
+  before_action :set_order, only: [:destroy]
```

Comment:
```suggestion
  before_action :set_order, only: :destroy
```
1 action bỏ ngoặc cũng đc ạ